src/spanningtrees/planar_maximally_filtered_graph.jl

```diff
@@ -0,0 +1,74 @@
+"""
+	planar_maximally_filtered_graph(g)
+
+Compete the Planar Maximally Filtered Graph (PMFG) of weighted graph `g`.
+
```

Comment:
Can we document the return type of this function here? I.e a `SimpleGraph`?

Reply:
There's a typo "compete"